x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -291,7 +293,7 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
     [containerElement, onSkipFocusBeforeEventsTable, onSkipFocusAfterEventsTable]
   );
 
-  const pageFiltersUpdateHandler = useCallback((newFilters: Filter[]) => {
+  const onFiltersChange = useCallback((newFilters: Filter[]) => {
```

Comment:
The world `Filters` can have many instances and I suggest you to rename this so that it is easier to understand  which filters are we talking about. Since these are essentially `controls`. May be `FilterControls` work. See what seem best to you. 

```suggestion
  const onFilterControlsChange = useCallback((newFilters: Filter[]) => {
```